src/main/java/com/capitalone/dashboard/collector/DefaultSonar6Client.java

```diff
@@ -200,8 +205,6 @@ public CodeQuality currentCodeQuality(SonarProject project) {
 
         } catch (ParseException e) {
             LOG.error("Could not parse response from: " + url, e);
-        } catch (RestClientException rce) {
-            LOG.error("Rest Client Exception: " + url + ":" + rce.getMessage());
```

Comment:
Since you already need to capture one exception at caller, why not handling all exceptions there, put all logic together

Reply:
Let me know if the changes work with this comment.  Just pushed more code.

Reply:
@benj58xu